aries-site/src/examples/components/namevaluelist/NameValueListAnatomy.js

```diff
@@ -42,66 +42,71 @@ const connections = [
   },
 ];
 
-const AnatomyBox = ({ background, id, visualId }) => (
-  <Box direction="row" gap="small">
-    <Box
-      id={visualId}
-      height="xxsmall"
-      width="xxsmall"
-      background={background}
-      round="xxsmall"
-    />
+const AnatomyGrid = ({ ...rest }) => (
+  <Grid
+    columns={['xxsmall', ['small', 'medium']]}
+    gap={{ column: 'small', row: 'medium' }}
+    justify="center"
+    {...rest}
+  />
+);
+
+const AnatomyBox = ({ background, id }) => {
+  const theme = useContext(ThemeContext);
+
+  return (
     <Box
       id={id}
-      height="xxsmall"
-      width="small"
+      fill="horizontal"
       background={background}
+      height={theme.global.edgeSize.medium}
       round="xxsmall"
     />
-  </Box>
-);
+  );
+};
 
-const CircleBox = ({ id, target }) => (
-  <Box
-    id={id}
-    align="center"
-    pad={{ horizontal: 'small', vertical: 'xxsmall' }}
-    round
-    background="background-front"
-  >
-    <Text weight="bold">{target}</Text>
-  </Box>
-);
+const Annotation = ({ id, target }) => {
+  const theme = useContext(ThemeContext);
+
+  return (
+    <Box
+      id={id}
+      align="center"
+      background="background-front"
+      border={{ color: 'border-weak' }}
+      height={theme.global.edgeSize.medium}
+      justify="center"
+      round
+      width={theme.global.edgeSize.medium}
+    >
+      <Text size="small" weight="bold">
+        {target}
+      </Text>
+    </Box>
+  );
+};
 
 export const NameValueListAnatomy = () => (
-  <Stack>
-    <Box gap="medium" pad={{ bottom: 'medium' }}>
-      <Box
-        margin={{ left: 'xlarge' }}
-        justify="between"
-        gap="large"
-        direction="row"
-        width="medium"
-      >
-        <CircleBox id={1} target="1" />
-        <CircleBox id={2} target="2" />
-      </Box>
-      <Box gap="medium" direction="row">
-        <AnatomyBox
-          id="name"
-          background="status-warning"
-          visualId="nameVisual"
-        />
-        <AnatomyBox id="value" background="purple!" visualId="valueVisual" />
-      </Box>
-      <Box
-        justify="between"
-        width="medium"
-        direction="row"
-      >
-        <CircleBox id={3} target="3" />
-        <CircleBox id={4} target="3" />
-      </Box>
+  <Stack margin={{ bottom: 'large' }}>
```

Comment:
Wondering if this diagram should be wrapped in the figure and have a figcaption?

Reply:
good call I can add that

Reply:
I think accessibility wise it should

Reply:
```suggestion
  <Stack margin={{ bottom: 'medium' }}>
```